jsk_hand/softhand_ros/README.md

```diff
@@ -0,0 +1,178 @@
+# softhand_ros
+
+ROS package for SoftHand
+
+## Note
+
+This package depends on the branch below.
+
+- [pazeshun/dynamixel_motor@gripper-v6-devel](https://github.com/pazeshun/dynamixel_motor/tree/gripper-v6-devel)
+
+## Installation
+
+### Workspace build
+
+```bash
+source /opt/ros/$ROS_DISTRO/setup.bash
+mkdir ~/softhand_ws/src -p
+cd ~/softhand_ws/src
+wget https://raw.githubusercontent.com/knorth55/softhand_ros/master/fc.rosinstall -O .rosinstall
+wstool up
+rosdep install --ignore-src --from-paths . -y -r -i
+cd ~/softhand_ws
+catkin build
+```
+
+### Udev installation
+
+```bash
+source ~/softhand_ws/devel/setup.bash
+roscd softhand_ros
+sudo cp udev/*.rules /etc/udev/rules.d
+sudo service udev reload
+sudo service udev restart
+```
+
+## How to use
+
+### Launch softhand
+
+#### For softhand v1
```

Comment:
please add more explanation of diference between v1 and v2

Reply:
I added.